docs/api/synchronization_library/barrier.md

```diff
@@ -23,26 +23,35 @@ cuda::std::barrier<> bb;
 cuda::barrier<cuda::thread_scope_block> c;
```

Comment:
What does this default/value/trivial initialization do for barrier? 

`std::barrier` only has the constructor `barrierbarrier(ptrdiff_t expected, CompletionFunction f = CompletionFunction());`

What is the `expected` count set to when using default/value/trivial initialization?